src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -485,6 +485,25 @@ public function preprocessEditorialLabel($cdbid, $settings)
         return $label;
     }
 
+    /**
+     * Format the publisher name for the UI
+     * @param string $publisher
+     * @return string
+     */
+    public function formatPublisherLabel($publisher)
+    {
+        switch (strtolower($publisher)) {
+            case 'uit':
+                return 'UiTinVlaanderen';
+                break;
```

Comment:
I think you can leave the break statements out since it are all returns. Unless php requires this?

Reply:
I copied it from another function.
@LucWollants  to the rescue

Reply:
This is also useful: https://stackoverflow.com/questions/1437461/php-coding-styles-return-in-switch-case

It comes down to `short syntax` vs `defensive programming` I think 🙂

Reply:
When doing an early `return` the `break` can be removed: https://stackoverflow.com/questions/6330114/do-you-need-break-in-switch-when-return-is-used

Reply:
Removed the break statements